JourneysSquad/JourneysSquad/View/AddBillView/Subviews/AddBillElementsView.swift

```diff
@@ -0,0 +1,94 @@
+import SwiftUI
+
+/// A view for adding a bill,
+struct AddBillElementsView: View {
+    // MARK: - Internal interface
+
+    var body: some View {
+        ScrollView {
+            LabelWithTextFieldView(
+                input: $sumOfBill,
+                labelText: howMuchFieldLabel,
+                placeholderText: howMuchFieldPlaceholder,
+                keyboardType: .numberPad
+            )
+            .padding()
+
+            LabelWithTextFieldView(
+                input: $spendingDescription,
+                labelText: whatIsItFieldLabel,
+                placeholderText: whatIsItFieldPlaceholder,
+                keyboardType: .default
+            )
+            .padding()
+
+            HStack(alignment: .top) {
+                LabelWithTextFieldView(
+                    input: $participantName,
+                    labelText: participantFieldLabel,
+                    placeholderText: participantFieldPlaceholder,
+                    keyboardType: .default
+                )
+
+                Spacer()
+
+                VStack {
+                    LabelWithTextFieldView(
+                        input: $paidAmount, labelText: paidFieldLabel,
+                        placeholderText: paidFieldPlaceholder,
+                        keyboardType: .numberPad,
+                        labelAlignment: .center,
+                        textFieldAlignment: .center
+                    )
+                    .frame(maxWidth: paidFieldMaxWidth)
+
+                    Button("") {}
+                        .buttonStyle(CircleWithPlusButton())
+                        .frame(
+                            width: circleWithPlusButtonFrameWidth,
+                            height: circleWithPlusButtonFrameHeight
+                        )
+                        .padding(.top)
+                }
+            }
+            .padding()
+
+            LabelWithDatePickerView(
+                date: $selectedDate,
+                labelText: "Date"
+            )
+            .padding()
+
+            Button(String.AppButtonLabelText.save) {}
+                .buttonStyle(BlackRoundedRectangleButton())
+                .padding()
+        }
+    }
+
+    // MARK: - Private interface
+
+    @State private var sumOfBill = ""
+    @State private var spendingDescription = ""
```

Comment:
Should we use a different name? I don't think the name is immediately clear.

For example:

```suggestion
    @State private var billDescription = ""
```

Reply:
Fixed